src/Core/src/Hosting/MauiAppBuilder.cs

```diff
@@ -3,16 +3,19 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging.Abstractions;
 using Microsoft.Maui.Dispatching;
 using Microsoft.Maui.LifecycleEvents;
+using System.Collections.Generic;
+using Microsoft.Extensions.Diagnostics.Metrics;
 
 namespace Microsoft.Maui.Hosting
 {
 	/// <summary>
 	/// A builder for .NET MAUI cross-platform applications and services.
 	/// </summary>
-	public sealed class MauiAppBuilder
+	public sealed class MauiAppBuilder : IHostApplicationBuilder
```

Comment:
This is mostly simple, but is adds some extra things.

Reply:
Look he s back :)